gold.mixin

```diff
@@ -0,0 +1,11 @@
+{
+    "build": {
+        "gold": {
+            "cmake-args": [
+                "-DCMAKE_EXE_LINKER_FLAGS=-fuse-ld=gold",
+                "-DCMAKE_SHARED_LINKER_FLAGS=-fuse-ld=gold",
+                "-DCMAKE_MODULE_LINKER_FLAGS=-fuse-ld=gold"
```

Comment:
I'd suggest lexically sorting JSON array elements unless the order is significant.

```suggestion
                "-DCMAKE_MODULE_LINKER_FLAGS=-fuse-ld=gold",
                "-DCMAKE_SHARED_LINKER_FLAGS=-fuse-ld=gold"
```

Reply:
It is not important afaik